java/arcs/sdk/android/storage/ServiceStore.kt

```diff
@@ -121,9 +123,12 @@ class ServiceStore<Data : CrdtData, Op : CrdtOperation, ConsumerData>(
     // So we need to create fresh instances when off() invoked
     private fun initChannel() {
         synchronized(this) {
-            channel?.let { it.cancel() }
-            channel = Channel(Channel.UNLIMITED)
-            channel!!.consumeAsFlow().onEach { it() }.launchIn(scope)
+            channel?.takeIf { !it.isClosedForSend }?.cancel()
+            channel = ConflatedBroadcastChannel<suspend () -> Unit>().also {
```

Comment:
what's the difference with a simple Channel?

Reply:
ConflatedBroadcastChannel only holds the latest item, the `.buffer()` we do below allows us to have some extra time to process incoming items without requiring us to specify `UNLIMITED` for the channel size.